plugins/quickcl.py

```diff
@@ -179,7 +182,12 @@ def __init__(self, microscope, main_app):
         self.light = main_data.light
         self.ccd = main_data.ccd
 
-        self.filename = model.StringVA("", setter=self._set_filename)
+        self.conf = get_acqui_conf()
```

Comment:
In QuickCL plugin, don't use/store the conf.last_extension. Just hard-code everything to ".png". That's because it's not a supported format for saving the standard data. Here, the main goal of using create_filename et al. is to reuse code, but not necessarily to share the name with the rest of the GUI. So the filename pattern and path can be stored, but not the format and extension.